
[2023-04-03 15:02:32] Traceback (most recent call last):
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 31, in <module>
    main()
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 16, in main
    cef.Initialize(settings)
  File "cefpython_py39.pyx", line 629, in cefpython_py39.Initialize
  File "settings.pyx", line 133, in cefpython_py39.SetApplicationSettings
Exception: Invalid appSettings key: allow_file_access_from_file_urls


[2023-04-03 15:02:36] Traceback (most recent call last):
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 31, in <module>
    main()
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 16, in main
    cef.Initialize(settings)
  File "cefpython_py39.pyx", line 629, in cefpython_py39.Initialize
  File "settings.pyx", line 133, in cefpython_py39.SetApplicationSettings
Exception: Invalid appSettings key: allow_file_access_from_file_urls


[2023-04-03 15:03:12] Traceback (most recent call last):
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 31, in <module>
    main()
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 16, in main
    cef.Initialize(settings)
  File "cefpython_py39.pyx", line 629, in cefpython_py39.Initialize
  File "settings.pyx", line 133, in cefpython_py39.SetApplicationSettings
Exception: Invalid appSettings key: allow_file_access_from_file_urls


[2023-04-03 15:04:55] Traceback (most recent call last):
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 34, in <module>
    main()
  File "c:\Users\Romaric\DataScience\EchecsVision\echec_vision\core\server\test.py", line 19, in main
    cef.Initialize(settings)
  File "cefpython_py39.pyx", line 629, in cefpython_py39.Initialize
  File "settings.pyx", line 133, in cefpython_py39.SetApplicationSettings
Exception: Invalid appSettings key: security


[2023-04-03 15:49:31] Traceback (most recent call last):
  File "lifespan_handler.pyx", line 116, in cefpython_py39.LifespanHandler_OnBeforeClose
  File "utils.pyx", line 50, in cefpython_py39.Debug
KeyboardInterrupt


[2023-04-03 15:54:58] Traceback (most recent call last):
  File "lifespan_handler.pyx", line 116, in cefpython_py39.LifespanHandler_OnBeforeClose
  File "utils.pyx", line 50, in cefpython_py39.Debug
KeyboardInterrupt

